schedule/security/usbguard.yaml

```diff
@@ -0,0 +1,17 @@
+name: usbguard
+description:    >
+    This is for usbguard test
+schedule:
+    - '{{bootloader_zkvm}}'
+    - '{{bootloader}}'
+    - boot/boot_to_desktop
+    - security/usbguard/usbguard
+conditional_schedule:
+    bootloader_zkvm:
+        ARCH:
+            s390x:
+                - installation/bootloader_zkvm
+    bootloader:
+        ARCH:
+            ppc64le:
+                - installation/bootloader
```

Comment:
Just a tiny comment:
you also could write
```
schedule:
    - '{{bootloader}}'
    - boot/boot_to_desktop
    - security/usbguard/usbguard
conditional_schedule:
    bootloader:
        ARCH:
            s390x:
                - installation/bootloader_zkvm
            ppc64le:
                - installation/bootloader
```

Reply:
Done.